### ISSUE TITLE
OracleDB fails to install on Windows 11 (wheel requires cpp tools)

### ISSUE THREAD

**Issue by tylerm007** [ApiLogicServer/ApiLogicServer-src#54]
pip install ApiLogicServer fails on Windows 11 (wheel for oracledb requires cpp tools)
<img width="627" alt="Screenshot 2024-04-15 at 2 04 20 PM" src="https://github.com/ApiLogicServer/ApiLogicServer-src/assets/7061078/a8464e18-fe4c-4abc-a66f-5587c051bf8c">

**Comment by tylerm007** [comment]
I have sent this to Oracle folks for comment.

**Comment by tylerm007** [comment]
From Oracle:
Christopher Jones
to me, Anthony, Veronica, Val | to me, Anthony, Veronica, Val
to me, Anthony, Veronica, Val

I see references to 312 in the screenshot, so presumably this is Python 3.12.

It was only in python-oracledb 1.4.2 that we released pre-built binaries for Python 3.12.  See https://python-oracledb.readthedocs.io/en/stable/release_notes.html#oracledb-1-4-2-october-2023
But overall we'd recommend using the latest version of python-oracledb which is 2.1.2.
Chris

**Comment by valhuber** [comment]
preliminary testing confirms resolution with the upgraded oracledb version.

You can use the preview:

```bash
python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple ApiLogicServer==10.03.76
```

**Comment by valhuber** [comment]
Fixed in 10.3.76
